index.html

```diff
@@ -113,6 +113,33 @@ <h2>A project for first-time contributions</h2>
         </div>
         <!-- ________ *TEMPLATE: MAKE A COPY* Contributor card END ________  -->
 
+        <!-- ________ *Jen card START ________  -->
+        <div class="card">
+          <p class="name">Jen</p>
+          <p class="contact">
+            <i class="fab fa-github"></i>
+            <a href="https://www.github.com" target="_blank">brite107</a>
+          </p>
+          <p class="about">I'm a computer science student at OSU and full-stack software engineer!
```

Comment:
I notice you have a line break in your file here... in HTML, this line break will not be preserved. If you intended that, then no worries -- nothing to do. 😄 

But if you wanted it to be a new line, you'd need to wrap each line in their own `<p>` tags, like so:

```
<p class="about">I'm a computer science student at OSU and full-stack software engineer!</p>
<p class="about">Taking a class in open source software.</p>
```

Some people will also use a `<br />` tag, but it's not really best-practice. Here's a link to [the MDN docs about `<br />`](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/br) where they give some guidance about this. ✌️

Reply:
Hi there! Oops, sorry about those oversights! 🫣
I fixed my github profile URL, and decided just one line was enough for about me. Thank you for your time!